plugins/completion/nvim-cmp/cmp-helpers.nix

```diff
@@ -1,54 +1,119 @@
-{ lib, pkgs, ... }@attrs:
+{ lib, pkgs, config, ... }@attrs:
+
+with lib;
+
 let
-  helpers = import ../../helpers.nix { lib = lib; };
-in with helpers; with lib;
-{
-  mkCmpSourcePlugin = { name, extraPlugins ? [], useDefaultPackage ? true, ... }: mkPlugin attrs {
-    inherit name;
-    extraPlugins = extraPlugins ++ (lists.optional useDefaultPackage pkgs.vimPlugins.${name});
-    description = "Enable ${name}";
-  };
+  helpers = import ../../helpers.nix { inherit lib config; };
 
-  pluginAndSourceNames = {
-    "luasnip" = "cmp_luasnip";
-    "snippy" = "cmp-snippy";
-    "ultisnips" = "cmp-nvim-ultisnips";
-    "vsnip" = "cmp-vsnip";
+  # remove nulls from list (Is there no lib function for this?)
```

Comment:
Use `lib.filter (k: !(isNull k))`

Reply:
I tried to find something like this. I was sure there is a function, but I could not find it 😅
Thanks :)